Crashes (segfaults) on Wayland

**Describe the bug**
Ever since the last update (from v1.3.12 to v1.3.14), I am fairly frequently getting crashes when running on Wayland (often when I am in a call). Although the crashes can occur randomly, I can reproduce it fairly consistently using the steps below.

**To Reproduce**
Steps to reproduce the behavior:
1. Launch in a terminal with the following command
   ```
   teams-for-linux --enable-features=UseOzonePlatform --ozone-platform=wayland
   ```
2. Quickly maximise the application before it has time to load fully.
3. It will crash and and I see this output in my terminal
   ```
   [ERROR] net::ERR_CONNECTION_REFUSED
   [1]    52878 segmentation fault (core dumped)  teams-for-linux --enable-features=UseOzonePlatform --ozone-platform=wayland 
   ```

**Expected behavior**
No crash or segfault like in v1.3.12.

**Screenshots**
n/a

**Desktop (please complete the following information):**
 - OS: Fedora 38 
 - Installation package: RPM
 - Version: 1.3.14
 - Desktop Environment: KDE Plasma 5.27.8
 - Kernel Version: 6.5.8-200.fc38.x86_64 (64-bit)
 - Graphics Platform: Wayland

**Additional context**
I've been experiencing a similar issue with VSCode which is also electron based under certain conditions (microsoft/vscode#184124) which has been blamed on this upstream electron issue (electron/electron#37531) and I think this might be the same issue. For now, I will stick with v1.3.12.

Hi folks, I reached this issue from #1016.

My _flatpak_ teams for linux crashes on my a couple of seconds into startup.

This is what I get if I run the command mentioned in the .desktop file in a shell:

```shell
andrea@andromeda:~> /usr/bin/flatpak run --branch=stable --arch=x86_64 --command=teams-for-linux --file-forwarding com.github.IsmaelMartinez.teams_for_linux @@u %U @@
[2:1130/102537.353402:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
LaunchProcess: failed to execvp:
xdg-settings
[2:1130/102537.552996:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:1130/102537.553048:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:1130/102537.583896:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:1130/102537.583921:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:1130/102537.583939:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[ERROR] net::ERR_CONNECTION_REFUSED
[47:1130/102542.973866:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
``` 

Am I in the right place? Is this what's being discussed here?

I'm running version 1.3.19 as a flatpak under opensuse Tumbleweed (kernel 6.6.2).
As a workaround I'm using the appimage which is thankfully working fine (version 1.3.23)

Thanks

--- a/com.github.IsmaelMartinez.teams_for_linux.appdata.xml
+++ b/com.github.IsmaelMartinez.teams_for_linux.appdata.xml
@@ -14,7 +14,7 @@
 	<url type="bugtracker">https://github.com/IsmaelMartinez/teams-for-linux/issues</url>
 	<launchable type="desktop-id">com.github.IsmaelMartinez.teams_for_linux.desktop</launchable>
 	<releases>
-		<release version="1.3.27" date="2023-12-21">
+		<release version="1.4.0" date="2023-12-21">
 			<description>
 				<ul>
 					<li>Electron version upgrade to 28.0.0</li>

--- a/com.github.IsmaelMartinez.teams_for_linux.appdata.xml
+++ b/com.github.IsmaelMartinez.teams_for_linux.appdata.xml
@@ -14,6 +14,14 @@
 	<url type="bugtracker">https://github.com/IsmaelMartinez/teams-for-linux/issues</url>
 	<launchable type="desktop-id">com.github.IsmaelMartinez.teams_for_linux.desktop</launchable>
 	<releases>
+		<release version="1.3.27" date="2023-12-21">
+			<description>
+				<ul>
+					<li>Electron version upgrade to 28.0.0</li>
+					<li>Update dependencies to latest versions</li>
+				</ul>
+			</description>
+		</release>
 		<release version="1.3.20" date="2023-11-14">
 			<description>
 				<ul>

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "teams-for-linux",
-  "version": "1.3.26",
+  "version": "1.3.27",
   "main": "app/index.js",
   "description": "Unofficial client for Microsoft Teams for Linux",
   "homepage": "https://github.com/IsmaelMartinez/teams-for-linux",
@@ -39,7 +39,7 @@
     "release": "electron-builder"
   },
   "dependencies": {
-    "@electron/remote": "^2.0.12",
+    "@electron/remote": "^2.1.1",
     "electron-is-dev": "2.0.0",
     "electron-store": "8.1.0",
     "electron-window-state": "5.0.3",
@@ -48,11 +48,11 @@
     "yargs": "17.7.2"
   },
   "devDependencies": {
-    "electron": "27.0.4",
-    "electron-builder": "24.6.4",
-    "@electron/fuses": "^1.6.2",
-    "eslint": "8.53.0",
-    "yarn": "1.22.19"
+    "@electron/fuses": "^1.7.0",
+    "electron": "28.0.0",
+    "electron-builder": "24.9.1",
+    "eslint": "8.56.0",
+    "yarn": "1.22.21"
   },
   "build": {
     "appId": "teams-for-linux",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "teams-for-linux",
-  "version": "1.3.27",
+  "version": "1.4.0",
   "main": "app/index.js",
   "description": "Unofficial client for Microsoft Teams for Linux",
   "homepage": "https://github.com/IsmaelMartinez/teams-for-linux",

